### ISSUE TITLE
Replace `TestedCell` with the extracted repository.

### ISSUE THREAD

**Issue by BrooklinJazz** [DockYard-Academy/beta_curriculum#185]
`TestedCell` has been extracted into it's own repository. https://github.com/BrooklinJazz/tested_cell.

We need to add the dependency to the setup section in every livebook.

```elixir
{:tested_cell, github: "BrooklinJazz/tested_cell"}
```

And ensure any existing calls to `TestedCell` continue to work. We may need to recreate them.

**Comment by Mraiih** [comment]
I can do it while I replace `Utils.feedback` with `TestedCell`

**Comment by BrooklinJazz** [comment]
That would be great! That will mean less work for replacing `Utils.feedback` if we do it now.

I recommend searching for all calls to `Mix.install` and adding the dependency to every livebook. Likely a clever find-and-replace could make this quick and easy rather than manually changing each one.

```elixir
Mix.install([
  {:kino, github: "livebook-dev/kino", override: true},
  {:kino_lab, "~> 0.1.0-dev", github: "jonatanklosko/kino_lab"},
  {:vega_lite, "~> 0.1.4"},
  {:kino_vega_lite, "~> 0.1.1"},
  {:benchee, "~> 0.1"},
  {:ecto, "~> 3.7"},
  {:math, "~> 0.7.0"},
  {:faker, "~> 0.17.0"},
  {:utils, path: "utils"}
])
```

Keep in mind the `Mix.install` sections are not `100%` consistent between live books (though perhaps they should be)

**Comment by Mraiih** [comment]
Do we still need `makeup` and `makeup_elixir` as dependencies of our livebooks since it's added to the TestedCell repository?

**Comment by BrooklinJazz** [comment]
Nope!

**Comment by BrooklinJazz** [comment]
However, we'll need to replace `Exercise` with `TestedCell` since that does rely on `makeup` + `makeup_elixir`.

**Comment by BrooklinJazz** [comment]
resolved #188
